kuma/static/stylus/components/wiki/section-edit.styl

```diff
@@ -0,0 +1,15 @@
+.section-edit {
+    float: right;
+    margin-left: 20px;
+    set-smaller-font-size();
+    opacity: 0;
+
+    @media $media-query-mobile {
+        display: none;
+    }
+
+    &:focus,
+    h2:hover & {
```

Comment:
Let's start with this. If it's effective, we might like to do a follow-on experiment to make the edit icons always visible on the section headers. Maybe just for signed-in users?

The goal is to make readers more aware that they can edit MDN, and to make it easier for them to do so.